pmm-app/tests/verifyQAN_test.js

```diff
@@ -0,0 +1,91 @@
+Feature('To verify and test the QAN Dashboard');
+
+Before(async (I, loginPage, qanPage, adminPage) => {
+  I.amOnPage(loginPage.url);
+  loginPage.login('admin', 'admin');
+  I.amOnPage(qanPage.url);
+  await I.waitForElement(qanPage.fields.iframe, 60);
+  adminPage.applyTimer('5m');
+  await I.switchTo(qanPage.fields.iframe);
+});
+
+Scenario(
+  'Open the QAN Dashboard and verify Filter groups exist @not-pr-pipeline',
+  async (I, adminPage, qanPage) => {
+    qanPage.waitForQANPageLoaded();
+    await qanPage.changeResultsPerPage(50);
+    qanPage.checkFilterGroups();
+  }
+);
+
+Scenario(
+  'Open the QAN Dashboard and verify Query Details section opened when user selects Query @not-pr-pipeline',
+  async (I, adminPage, qanPage) => {
+    qanPage.waitForQANPageLoaded();
+    qanPage._selectDetails(2);
+  }
+);
+
+Scenario('Open the QAN Dashboard and verify pagination @not-pr-pipeline', async (I, adminPage, qanPage) => {
```

Comment:
Let's try to apply coherent formatting as in previous scenarios.

Reply:
Reformatted and cleaned everything, there is still a lot of things to improve, but it's for future tasks